dvc/remote/ssh/__init__.py

```diff
@@ -10,6 +10,7 @@
 from urllib.parse import urlparse
 
 from funcy import memoize, wrap_with, silent, first
+import paramiko
```

Comment:
Let's move this under `RemoteSSH.__init__` as a dynamic import.

Reply:
OK, I'll do that.

Reply:
`RemoteSSH._load_user_ssh_config` is a static method and uses paramiko, this means that it can be called without executing the import inside the `RemoteSSH.__init__`. Should I place the import directly inside the class body? Meaning:
```python
class RemoteSSH(RemoteBASE):
    import paramiko
```

Reply:
@gio8tisu Doesn't seem to work, judging by the tests. I would just keep it simple: just add `import paramiko` in `__init__` as initially requested.